packages/@glimmer-workspace/build/lib/config.js

```diff
@@ -356,7 +381,20 @@ export class Package {
    */
   replacements(env) {
     return env === 'prod'
-      ? [importMeta]
```

Comment:
been having doubts about this plugin because import.meta.env.PROD checks were being ignored / not DCEing